src/main/java/com/afornalik/ox/board/Board.java

```diff
@@ -4,33 +4,71 @@
 import java.util.Map;
 import java.util.TreeMap;
 
+/**
+ * Board class is responsible for hold information about board,
+ * also this class make some basic operations.
+ *
+ * @author Andrzej Fornalik
+ * @version 1.0
```

Comment:
Jak tam z wersjonowaniem? Rozumiem, że przy 1.0 to już finalny produkt, tak?

Reply:
Słusznie, nie pomyślałem. zmienione.